Formula/sickchill.rb

```diff
@@ -0,0 +1,430 @@
+class Sickchill < Formula
+  include Language::Python::Virtualenv
+
+  desc "Media library mega tool, packaged into a homebrew bottle with a service"
+  homepage "https://sickchill.github.io"
+  url "https://files.pythonhosted.org/packages/ba/75/d27b26bdac593423026a8d606551de134f745fb36ec991ff0a70d4dfb2a0/sickchill-2023.5.30.tar.gz"
+  sha256 "be49f0d6c01f479cb0f066e88582c4971b683a11d50149ffe81b1bdc4447c2ab"
```

Comment:
It would be good to add the `license` as well.

Reply:
Yep, I added it on my tap, I will add it here as soon as that builds